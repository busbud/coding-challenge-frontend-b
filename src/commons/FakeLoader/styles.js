import styled from "styled-components";

export const Loader = styled.div`
  @keyframes background_dance {
    from {
      background-color: var(--light-gray);
    }

    to {
      background-color: var(--gray);
    }
  }

  width: 100%;
  height: 12px;
  animation: background_dance 0.4s infinite alternate-reverse;
  border-radius: 10px;
  display: block;

  &:not(:last-of-type) {
    margin-bottom: 8px;
  }
`;
