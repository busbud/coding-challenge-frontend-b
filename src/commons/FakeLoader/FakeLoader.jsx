import React from "react";

import { Loader } from "./styles";

const FakeLoader = ({ ...otherProps }) => <Loader {...otherProps} />;

export default FakeLoader;
