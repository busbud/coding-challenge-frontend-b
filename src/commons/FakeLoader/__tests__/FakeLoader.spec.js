import React from "react";
import { render } from "@testing-library/react";

import FakeLoader from "../";

describe("<FakeLoader />", () => {
  it("should render properly", () => {
    const { container } = render(<FakeLoader />);
    expect(container).toMatchSnapshot();
  });
});
