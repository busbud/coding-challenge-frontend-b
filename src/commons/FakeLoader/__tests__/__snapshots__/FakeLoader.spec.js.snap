// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FakeLoader /> should render properly 1`] = `
<div>
  <div
    class="sc-bdfBwQ chHSNf"
  />
</div>
`;
